libcontainer/cgroups/fs/memory.go

```diff
@@ -282,21 +282,20 @@ func getPageUsageByNUMA(path string) (cgroups.PageUsageByNUMA, error) {
 		line := scanner.Text()
 		columns := strings.SplitN(line, " ", maxColumns)
 		for i, column := range columns {
-			byNode := strings.SplitN(column, "=", 2)
+			key, val, ok := strings.Cut(column, "=")
 			// Some custom kernels have non-standard fields, like
 			//   numa_locality 0 0 0 0 0 0 0 0 0 0
 			//   numa_exectime 0
-			if len(byNode) < 2 {
-				if i == 0 {
-					// Ignore/skip those.
-					break
-				} else {
+			if !ok {
+				if i != 0 {
 					// The first column was already validated,
 					// so be strict to the rest.
 					return stats, malformedLine(path, file, line)
 				}
+
+				// Ignore/skip those.
+				break
```

Comment:
With this change, it is no longer clear that "Ignore/skip those" is related to "Some custom kernels have non-standard fields".

Reply:
i changed it